frontend/src/metabase-lib/queries/utils/structured-query-table.ts

```diff
@@ -50,6 +50,7 @@ function getFieldsForSourceQueryTable(
     const virtualField = createVirtualField({
       ...column,
       id,
+      source: "fields",
```

Comment:
This is what we missed from https://github.com/metabase/metabase/pull/25267. I'm not sure if this code is intended to fix it or not. But after that PR, the field source for **Count** would be `aggregation` which sounds more correct to me, but would produce the bug. Forcing the field source to be `fields` would follow the old behavior, thus fixing the regression.